e2e-tests/insurance/cypress/e2e/journeys/policy/change-your-answers/broker/change-your-answers-change-broker-no-to-yes.spec.js

```diff
@@ -0,0 +1,75 @@
+import { summaryList } from '../../../../../../../pages/shared';
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance/policy';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import checkSummaryList from '../../../../../../../commands/insurance/check-policy-summary-list';
+
+const {
+  USING_BROKER: FIELD_ID,
+  BROKER_DETAILS: {
+    NAME,
+    EMAIL,
+    FULL_ADDRESS,
+  },
+} = POLICY_FIELD_IDS;
+
+const {
+  ROOT,
+  POLICY: { BROKER_CHANGE, CHECK_YOUR_ANSWERS },
+} = INSURANCE_ROUTES;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Policy - Change your answers - Broker - No to yes - As an exporter, I want to change my answers to the broker section', () => {
```

Comment:
can we make these consistent please? different format to e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-no-to-yes.spec.js

Reply:
Good spot. Have updated to be aligned.

We need to go through other "change your answers" E2E tests, not for this PR